Goal[] goals
float32[] blends
---
bool success
bool configuration_change